docs/source/release_notes/release-notes.rst

```diff
@@ -52,7 +52,7 @@ develop
 .. <<<<------------------------------------------------------------------------------------------------------------->>>>
 
 =======
-v0.60.0
+v0.60.1
 =======
  
 12 October 2017
```

Comment:
One additional change I'd suggest: the 3rd entry for 0.60.1 (changing CassClientPool to CassClientPoolImpl etc.) mentions `make the implementation of the above feasible`, which I think should be changed to `make the implementation of startup independence feasible` or something along those lines. This release note probably used to be just after the Async Initializer changes, but then we added the TimeLock DB Persister work.